proto/src/warden.proto

```diff
@@ -10,6 +10,12 @@ message RangeId {
 message HostInfo {
     string identity = 1;
     string zone = 2;
+    // The epoch is used to handle out of order registration/deregistration.
```

Comment:
would there be a problem if a range server connects twice with the same epoch value?

Reply:
Good point. I hadn't considered that edge case. Now I have updated the code in `register_range_server` to reject requests with the same or older epoch values. I think in practice this should rarely happen and having the client retry in that case is the right call. The client code currently reads the epoch value for every retry to connect.